web/unit_tests/test_utils.js

```diff
@@ -30,35 +46,73 @@ var setupKMW = function(ui) {
    */
   window.setTimeout(function() {
     initFunc();
-  }, 5);
+  }, 8);
 }
 
-var setupScript = function(src) {
+var setupScript = function(src, done, timeout, uiInitCheck) {
   var Lscript = document.createElement('script');
   Lscript.charset="UTF-8";        // KMEW-89
   Lscript.type = 'text/javascript';
   Lscript.async = false;
+  if(done) {
+    Lscript.onload = initTimer(done, timeout, uiInitCheck);
+  }
 
   Lscript.src = src;
 
   return Lscript;
 }
 
 var teardownKMW = function() {
+  keyman['shutdown']();
   delete window.keyman;
   window["keyman"] = undefined;
```

Comment:
This isn't the best way to remove a property from an object. See https://stackoverflow.com/questions/208105/how-do-i-remove-a-property-from-a-javascript-object

Reply:
So... exactly what I did above, but without the `window["keyman"] = undefined` line after it?

Reply:
Correct 👍